crates/context/src/lib.rs

```diff
@@ -354,7 +354,10 @@ impl ContextManager {
 
         let context_exists = handle.has(&ContextMetaKey::new(context_id))?;
         let mut config = if !context_exists {
-            let proxy_contract = self.get_proxy_contract(context_id).await?;
+            let proxy_contract = self
+                .get_proxy_contract(context_id)
+                .await
+                .unwrap_or_else(|_| "".to_owned());
```

Comment:
if there's an error, we should handle it

Reply:
Error is that it does not exists.

Reply:
if it doesn't exist, then there's a deeper issue, the config contract didn't record the context, and if that's the case, we shouldn't be here at all

Reply:
> It exists on owner side but it does not exists for invited peer

`get_proxy_contract` calls the contract, and it would only return that error if the context doesn't exist on the config contract

Reply:
Invited peer joined context but catchup failed.

Reply:
in description

Reply:
doesn't explain why ignoring the error is the solution

the `propose` method wasn't found because it was calling the wrong contract, which this PR fixes

but this line, you said was because the context didn't exist, in which case, we shouldn't be ignoring the error but figuring out why that's the case.

but if the error no longer happens, then we don't need this line, and any errors that occur here should be handled.